test/langtools/tools/javac/reflect/CodeReflectionTester.java

```diff
@@ -61,6 +69,28 @@ static void error(String msg, Object... args) {
         System.err.println("error: " + String.format(msg, args));
     }
 
+    static void checkModel(Member member, String found, IR ir) {
```

Comment:
This new method allows the harness to print the model we found on the compiled class even in the face of an error when parsing the @IR annotation. This is very handy when bulk-updating the tests, as it means that parser errors will cause the harness to just dump the "expected" model for all the methods in the class (instead of throwing on the first parser error).